src/components/Portfolio/ProjectDescription.jsx

```diff
@@ -0,0 +1,10 @@
+import { CommonButton } from "../index";
+
+export function ProjectDescription({ description, url, button }) {
+	return (
+		<>
+			<p>{description}</p>
+			<CommonButton url={url} content={button} style={{ width: "200px" }} />
```

Comment:
```suggestion
			<CommonButton url={url} content={button} />
```
Maybe this style is unnecessary because `<CommonButton />` has the width property already.

Reply:
Thanks for finding the unnecessary code.
I'll delete it!